src/main/java/picard/cmdline/CommandLineParser.java

```diff
@@ -235,9 +238,9 @@ private String getUsagePreamble() {
 
         if (null != this.programVersion && 0 < this.programVersion.length()) {
             usagePreamble += "Version: " + getVersion() + "\n";
+            usagePreamble += "HTSJDK Version: " + getHtsjdkVersion() + "\n";
```

Comment:
Alternatively, you should probably update the version reporting on line 325 and 497 as well as the definition of `CommandLineProgram.getVersion()`